xml/cleanup_and_order_mads.xsl

```diff
@@ -0,0 +1,175 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="1.0" 
+  xmlns:mads="http://www.loc.gov/mads/v2"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <xsl:output method="xml" version="1.0" encoding="UTF-8" indent="yes" media-type="text/xml"/>

```

Comment:
I think I'll just remove the  media-type="text/xml" as it doesn't appear
to be used in our other xslts.
dm

>>> Jared Whiklo <notifications@github.com> 06/10/15 12:49 PM >>>
> @@ -0,0 +1,175 @@
> +<?xml version="1.0" encoding="UTF-8"?>
> +<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
version="1.0" 
> +  xmlns:mads="http://www.loc.gov/mads/v2"
> +    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
> +  <xsl:output method="xml" version="1.0" encoding="UTF-8"
indent="yes" media-type="text/xml"/>

Should media-type be "application/xml" now?

---
Reply to this email directly or view it on GitHub:
https://github.com/Islandora/islandora_solution_pack_entities/pull/77/files#r32132359